examples/src/main/java/com/hedera/sdk/examples/advanced/TransferCryptoToExchange.java

```diff
@@ -0,0 +1,77 @@
+package com.hedera.sdk.examples.advanced;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.hedera.sdk.Client;
+import com.hedera.sdk.HederaException;
+import com.hedera.sdk.Transaction;
+import com.hedera.sdk.account.AccountCreateTransaction;
+import com.hedera.sdk.account.CryptoTransferTransaction;
+import com.hedera.sdk.crypto.ed25519.Ed25519PrivateKey;
+import com.hedera.sdk.examples.ExampleHelper;
+
+public class TransferCryptoToExchange {
+    // the exchange should possess this key, we're only generating it for demonstration purposes
+    private static final Ed25519PrivateKey exchangeKey = Ed25519PrivateKey.generate();
+
+    // this is the only key we should actually possess
+    private static final Ed25519PrivateKey userKey = Ed25519PrivateKey.generate();
+
+    private static final Client client = ExampleHelper.createHederaClient();
+
+    public static void main(String[] args) throws HederaException, InvalidProtocolBufferException {
+        var transferAmount = 10_000;
+
+        // the exchange creates an account for the user to transfer funds to
+        var exchangeAccountReceipt = new AccountCreateTransaction(client).setKey(exchangeKey.getPublicKey())
+            // the exchange only accepts transfers that it validates through a side channel (e.g. REST API)
+            .setReceiverSignatureRequired(true)
+            .executeForReceipt();
+
+        var exchangeAccountId = exchangeAccountReceipt.getAccountId();
+
+        if (exchangeAccountId == null) {
```

Comment:
I would use `Objects.requireNonNull` here. This shouldn't ever be null if the receipt comes back OK